moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamRoleMultiselect.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+
+import {
+  Checkbox,
+  FormControl,
+  FormHelperText,
+  Input,
+  ListItemText,
+  MenuItem,
+  Select,
+  Typography,
+} from "@mui/material";
+import makeStyles from '@mui/styles/makeStyles';
+import { useGridApiContext } from "@mui/x-data-grid-pro";
+
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+  },
+  noLabel: {
+    marginTop: theme.spacing(3),
+  },
+  infoIcon: {
+    color: theme.palette.action.disabled,
+  },
+}));
+
+const ProjectTeamRoleMultiselect = ({ id, field, roles, value }) => {
+  const rolesArray = value.map((role) => role.project_role_id);
+  const [selectedValues, setSelectedValues] = React.useState(rolesArray || []);
+
+  const classes = useStyles();
+  const apiRef = useGridApiContext();
+
+  const handleChange = (event) => {
+    const valueIds = event.target.value;
+    const newValue = roles.filter((role) => valueIds.includes(role.project_role_id));
+    const rolesArray = newValue.map((role) => role.project_role_id);
+    setSelectedValues(rolesArray);
+    apiRef.current.setEditCellValue({
+      id,
+      field,
+      value: newValue,
+    });
+  };
+
+  return (
+    <FormControl variant="standard" className={classes.formControl}>
+      <Select
+        variant="standard"
+        style={{ minWidth: "8em" }}
```

Comment:
i know a lot of this code is ported from the previous version, but maybe an opportunity here to switch from `em` to `rem`?

Reply:
looks great to me! thanks for updating. 🙏